app/api/admin/users/route.ts

```
@@ -0,0 +1,89 @@
+    const { searchParams } = new URL(request.url)
+    const page = parseInt(searchParams.get('page') || '1')
+    const limit = parseInt(searchParams.get('limit') || '20')
+    const search = searchParams.get('search') || ''
+    const role = searchParams.get('role') || ''
+    const isBanned = searchParams.get('isBanned') || ''
+
+    const db = await getDatabase()
+    const skip = (page - 1) * limit
+
+    // Build query
+    const query: any = {}
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)